dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/provisioning/ProvisioningFeatureDefinition.java

```diff
@@ -165,41 +168,69 @@ public interface ProvisioningFeatureDefinition extends FeatureDefinition {
     boolean isCapabilitySatisfied(Collection<ProvisioningFeatureDefinition> supplyingDefs);
 
     /**
-     * Answer the platform of this feature.
+     * Answer the names of the platform (or platforms) of this feature.
+     *
+     * Platform names are stored as header attribute WLP-Platform.
      *
-     * The feature platform is stored as a header attribute.
+     * When more than one platform name is specified, each should be a
+     * different version of the same base platform.
      *
-     * @return The platform of this feature.
+     * @return The platform names of this feature.
      */
     List<String> getPlatforms();
 
     /**
      * Tell if this is a versionless feature.
      *
-     * @return
+     * Versionless features are public, and have at least on WLP-Platform
+     * value.
```

Comment:
Versionless features are public features that do not have a version. They do not have a wlp-platform value.